### PR TITLE
Fix thumbnail url parsing

### PR DESCRIPTION
**Pull request by schwers**
1X api client used to strip out `post.thumbnail`s that weren't urls, and were hints at how to render. This makes it easier to do our fancier preview image generation and makes the api a little more consistent. This restores that behavior in 2X

### CODE REVIEW

**Comment by schwers** [comment]
👓 @phil303 or @uzi

File: models2/PostModel.es6.js [hunk 1]
```diff
@@ -5,6 +5,14 @@ import replyable from './mixins/replyable';
 
 const T = RedditModel.Types;
 
+const cleanThumbnail = thumbnail => {
+  if (thumbnail && (thumbnail === 'default' || thumbnail === 'self' || thumbnail === 'nsfw')) {
```
Comment (uzi) [inline models2/PostModel.es6.js]: Make an array with `['default', 'self', 'nsfw']` and `ARRAY.indexOf(thumbnail) !== -1`?

Comment (phil303) [inline models2/PostModel.es6.js]:
```js
const cleanThumbnail = thumbnail => {
  const IGNORE = ['default', 'self', 'nsfw'];   // or use a set
  return thumbnail && IGNORE.indexOf(thumbnail) > -1 ? '' : thumbnail;
};
```

Comment (uzi) [inline models2/PostModel.es6.js]: Let me 🔕  that one

**Comment by schwers** [comment]
💇

**Comment by phil303** [comment]
:fish: :fish:

File: models2/PostModel.es6.js [hunk 2]
```diff
@@ -130,6 +136,10 @@ export default class PostModel extends RedditModel {
         }],
       };
     },
+
+    thumbnail(data) {
```
Comment (phil303) [inline models2/PostModel.es6.js]: actually, shouldn't this be `thumbnail`

**Comment by uzi** [comment]
:fish:

**Pull request merged**